pkg/aws/manager_ec2_route_table.go

```diff
@@ -0,0 +1,104 @@
+package aws
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws/awserr"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/resourcegroupstaggingapi"
+	"github.com/integr8ly/cluster-service/pkg/clusterservice"
+	"github.com/integr8ly/cluster-service/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	loggingKeyRouteTable = "route-table-id"
+
+	resourceTypeRouteTable = "ec2:route-table"
+)
+
+var _ ClusterResourceManager = &RouteTableManager{}
+
+type RouteTableManager struct {
```

Comment:
Do we need this Manager, shouldn't the route table be cleaned up when the VPC is deleted?

Reply:
no the sdk doesn't work the same as the console..

Reply:
Wow that sucks, so it provisions the route table, but we have to manually remove the route table :man_shrugging:

Reply:
yea sucks